docs/pipelines/ner/behaviors/alcohol.md

```diff
@@ -0,0 +1,176 @@
+# Alcohol consumption
+
+The `eds.alcohol` pipeline component extracts mentions of alcohol consumption. It won't match occasionnal consumption, nor acute intoxication.
+
+??? info "Details of the used patterns"
+    <!-- no-check -->
+    ```python
+    # fmt: off
+    --8<-- "edsnlp/pipelines/ner/behaviors/alcohol/patterns.py"
+    # fmt: on
+    ```
+
+## Extensions
+
+On each span `span` that match, the following attributes are available:
+
+- `span._.status_`: set to either
```

Comment:
Could we use standard identifiers, like `status` here ? I see your point (I think) with spacy's label_ for instance but I think this would be more consistent with other extensions of edsnlp

Reply:
Actually `_.status` is also used 
> *Matches have an associated `_.status` attribute taking the value `0`, `1`, or `2`. A corresponding `_.status_` attribute stores the human-readable status, which can be component-dependent. See each component documentation for more details.*  

So the rationale is mainly that `status` is 0,1 or 2 regardless of the pipe, whereas `status_` is pipe-dependent. WDYT ?

Reply:
Now available in the `detailled_status` extension